### DISCUSSION TITLE
set maximum number of audio channels for mc routing

### DISCUSSION THREAD

**Discussion by the-steech** [ppooll-dev/ppooll#59] [Polls]
should we set the maximum property in any "number of channels" selection across ppooll acts/ll.blues?

for example: ho_st "chan" ll_num slider, i think the max should be set to 1024 channels

**Comment by chausch** [comment]
@the-steech i think that's a typical post for the discussion section of this repository. do you mind transferring it over there?

**Comment by chausch** [comment]
possibly "ideas" sub-section 🤔

**Comment by chausch** [comment]
@the-steech 
ah, i just noticed it's possible to convert issues to discussions. however by clicking too fast i put it in the wrong category. should be in ideas, not polls.

**Comment by chausch** [comment]
anyway i think 1024 could be veeeery much ok as limit. 😊 even if you have a wave field synthesis setup available you will likely not hit the limit ever. 😅
but i don't have a string opinion here. do you think having no limit will lead to crashes by accidentially incresing the count too high?

**Comment by the-steech** [comment]
@chausch i got that number from the maximum number of audio in/out channels in Max "audio status" settings.  I probably won't use more than 16 at a time.  Right now live.ppooll only supports 16... but we can worry about that later :)

**Reply by chausch** [reply]
sounds good!

**Comment by klausfilip** [comment]
by testing, i found that 1024 is the maximum number for a mc-signal. did not find it in the documentation..
